packages/integration-tests/src/test-cdk-projects/auth-cdk/test-cdk-project-auth-cdk-stack.ts

```diff
@@ -0,0 +1,31 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { AmplifyAuth, AuthProps } from '@aws-amplify/auth-construct-alpha';
+import * as process from 'process';
+
+export class TestCdkProjectAuthCdkStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    // Inject stack name from env to 1. know reference without lookup 2. ensure uniqueness.
+    super(scope, process.env.TEST_STACK_NAME, props);
```

Comment:
it's kinda tomatoes tomātoes, but maybe better to pass this in as a CDK context value rather than an env var? Defining as CDK context gives me a much better idea about where to look for where this value is set vs an env var where it could be set anywhere.